warre/inventory/inventory.html

```diff
@@ -23,16 +23,110 @@
 			crossorigin="anonymous"></script>
 
 	<!-- internal resource links -->
+	<!--
 	<script src="../js/global_on_page_load.js"></script>
 	<script src="../js/banner.js"></script>
 	<script src="../js/authentication.js"></script>
+	-->
+	<script src="js.cookie.js"></script>
 	<link href="inventory.css" type="text/css" rel="stylesheet">
 
 	<!-- initialize page -->
 	<script>
 		jQuery(document).ready(function(){
```

Comment:
I would really like this to be it's own JS file if possible, inline js is annoying.